openhtf/exe/__init__.py

```diff
@@ -128,11 +127,11 @@ def _RunTestLoop(self):
       exit_stack.callback(lambda: setattr(self, '_current_exit_stack', None))
 
       plug_manager = self._MakePlugManager(suppressor)
-      phase_executor = self._MakePhaseExecutor()
+      executor = self._MakePhaseExecutor()
```

Comment:
It's weird to have a variable called `executor` passed around inside a `TestExecutor()` that's actually a `PhaseExecutor()`